### PR TITLE
Now you can sign up and sign in on the system

### PR DESCRIPTION
**Pull request by amandasavluchinske**
Also added a homepage and fixed an issue in which Pokémon were being ordered crescently instead of in the order of the user's input.

### CODE REVIEW

**Comment by amandasavluchinske** [comment]
@chocoelho

**Comment by amandasavluchinske** [comment]
should be ok now @chocoelho

File: templates/accounts/register.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    <form class='inputs--auth-item' method="post">
+        {% csrf_token %}
+        <fieldset>
+            {{ form.non_field_errors }} {% for field in form %}
+            <div>
+                {{ field.label_tag }} {{ field }} {{ field.errors }}
+            </div>
+            {% endfor %}
+            <button type="submit" class="button--battle-submit">Entrar</button>
```
Comment (chocoelho) [inline templates/accounts/register.html]: pt-br text here

File: templates/accounts/login.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    <form class='inputs--auth-item' method="post">
+        {% csrf_token %}
+        <fieldset>
+            {{ form.non_field_errors }} {% for field in form %}
+            <div>
+                {{ field.label_tag }} {{ field }} {{ field.errors }}
+            </div>
+            {% endfor %}
+            <button type="submit" class="button--battle-submit">Entrar</button>
```
Comment (chocoelho) [inline templates/accounts/login.html]: pt-br text here

File: battles/helpers.py [hunk 3]
```diff
@@ -64,11 +71,11 @@ def make_sure_players_cant_choose_the_same_pokemon(battle, user, pokemon_list):
         trainer=user
     )
     your_pokemon = get_your_team.pokemons.all()
-    if not your_pokemon.exists():
+    if your_pokemon.count() == 0:
         your_pokemon = pokemon_list
     get_their_team = battle.teams.exclude(trainer=user).first()
     their_pokemon = get_their_team.pokemons.all()
-    if not their_pokemon.exists():
+    if their_pokemon.count() == 0:
```
Comment (chocoelho) [inline battles/helpers.py]: any specific reason to use `count` as checkage instead of `exists`?

File: battles/helpers.py [hunk 4]
```diff
@@ -35,10 +35,17 @@ def get_battle_team_by_trainer(battle, trainer):
...
+
+
 # Checks if the user has already chosen their team
 def check_if_user_has_not_chosen_their_team(battle, user):
-    trainer_battle_team = get_battle_team_by_trainer(battle=battle, trainer=user)
-    return not trainer_battle_team.pokemons.exists()
+    get_team = get_battle_team_by_trainer(battle=battle, trainer=user)
+    has_pokemon = get_team.pokemons.count()
+
+    return has_pokemon == 0
```
Comment (chocoelho) [inline battles/helpers.py]: same about using `count` instead of `exists` here

**Pull request merged**